packages/jose/jose.0.9.0/opam

```diff
@@ -0,0 +1,54 @@
+opam-version: "2.0"
+synopsis: "JOSE implementation for OCaml and ReasonML"
+description:
+  "JavaScript Object Signing and Encryption built ontop of pure OCaml libs"
+maintainer: ["ulrik.strid@outlook.com"]
+authors: ["Ulrik Strid"]
+license: "MIT"
+homepage: "https://ulrikstrid.github.io/reason-jose"
+doc: "https://ulrikstrid.github.io/reason-jose"
+bug-reports: "https://github.com/ulrikstrid/reason-jose/issues"
+depends: [
+  "ocaml" {>= "4.08.0"}
+  "base64" {>= "3.3.0"}
+  "dune" {>= "2.8"}
+  "eqaf" {>= "0.7"}
+  "mirage-crypto" {>= "0.10.0"}
+  "x509" {>= "0.13.0"}
+  "cstruct" {>= "6.0.0"}
+  "astring"
+  "yojson" {>= "1.6.0"}
+  "zarith"
+  "ptime"
+  "mirage-crypto-rng" {with-test & < "0.11.0"}
+  "containers" {with-test}
+  "bisect_ppx" {with-test}
+  "alcotest" {with-test}
+  "junit" {with-test}
+  "junit_alcotest" {with-test}
+  "odoc" {with-doc}
+]
```

Comment:
```suggestion
]
conflicts: [
  "result" {< "1.5"}
]
```
```
#=== ERROR while compiling jose.0.9.0 =========================================#
# context              2.2.0~alpha~dev | linux/x86_64 | ocaml-base-compiler.4.13.1 | pinned(https://github.com/ulrikstrid/reason-jose/releases/download/v0.9.0/jose-v0.9.0.tbz)
# path                 ~/.opam/4.13/.opam-switch/build/jose.0.9.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p jose -j 255 @install
# exit-code            1
# env-file             ~/.opam/log/jose-7-901105.env
# output-file          ~/.opam/log/jose-7-901105.out
### output ###
# File "/home/opam/.opam/4.13/lib/biniou/biniou.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
#       ocamlc jose/.Jose.objs/byte/jose__Jwk.{cmi,cmo,cmt} (exit 2)
# (cd _build/default && /home/opam/.opam/4.13/bin/ocamlc.opt -w -40 -g -bin-annot -I jose/.Jose.objs/byte -I /home/opam/.opam/4.13/lib/asn1-combinators -I /home/opam/.opam/4.13/lib/astring -I /home/opam/.opam/4.13/lib/base/caml -I /home/opam/.opam/4.13/lib/base64 -I /home/opam/.opam/4.13/lib/bigarray-compat -I /home/opam/.opam/4.13/lib/biniou -I /home/opam/.opam/4.13/lib/bytes -I /home/opam/.opam/4.13/lib/cstruct -I /home/opam/.opam/4.13/lib/domain-name -I /home/opam/.opam/4.13/lib/easy-format -I /home/opam/.opam/4.13/lib/eqaf -I /home/opam/.opam/4.13/lib/eqaf/bigstring -I /home/opam/.opam/4.13/lib/eqaf/cstruct -I /home/opam/.opam/4.13/lib/fmt -I /home/opam/.opam/4.13/lib/gmap -I /home/opam/.opam/4.13/lib/logs -I /home/opam/.opam/4.13/lib/mirage-crypto -I /home/opam/.opam/4.13/lib/mirage-crypto-ec -I /home/opam/.opam/4.13/lib/mirage-crypto-pk -I /home/opam/.opam/4.13/lib/mirage-crypto-rng -I /home/opam/.opam/4.13/lib/nocrypto -I /home/opam/.opam/4.13/lib/parsexp -I /home/opam/.opam/4.13/lib/pbkdf -I /home/opam/.opam/4.13/lib/ppx_deriving/runtime -I /home/opam/.opam/4.13/lib/ppx_sexp_conv/runtime-lib -I /home/opam/.opam/4.13/lib/ptime -I /home/opam/.opam/4.13/lib/result -I /home/opam/.opam/4.13/lib/rresult -I /home/opam/.opam/4.13/lib/seq -I /home/opam/.opam/4.13/lib/sexplib -I /home/opam/.opam/4.13/lib/sexplib0 -I /home/opam/.opam/4.13/lib/stdlib-shims -I /home/opam/.opam/4.13/lib/x509 -I /home/opam/.opam/4.13/lib/yojson -I /home/opam/.opam/4.13/lib/zarith -no-alias-deps -open Jose__ -o jose/.Jose.objs/byte/jose__Jwk.cmo -c -impl jose/Jwk.ml)
# File "jose/Jwk.ml", line 836, characters 7-17:
# 836 |     |> Result.map (fun key -> Ed25519_pub (make_jwk key))
#              ^^^^^^^^^^
# Error: Unbound value Result.map
```